src/main/dialogs/quickSetStrip/qssDialog.js

```diff
@@ -187,21 +187,24 @@ fluid.defaults("gpii.app.qss", {
     }
 });
 
-
+/**
+ * Represents a group of setting data from which we using only the buttonTypes array
+ * @typedef {Object} ButtonList
+ * @property {Array} [buttonTypes] Array from diffent button types
```

Comment:
As it it a typedef, I think it would be better if there are a few more properties